platform-modules/mimetype-manager/src/main/scala/org/sunbird/cloudstore/StorageService.scala

```diff
@@ -101,4 +103,67 @@ class StorageService {
               ""
         }
     }
+
+  def getMimeType(fileName: String): String = { // 1. first use java's built-in utils
+    val mimeTypes = URLConnection.getFileNameMap
+    var contentType = mimeTypes.getContentTypeFor(fileName)
+    // 2. nothing found -> lookup our in extension map to find types like ".doc" or ".docx"
+    if (contentType == null) {
+      val extension = fileName.substring(fileName.lastIndexOf('.') + 1, fileName.length)
+      contentType = fileExtensionMap.getOrDefault(extension, "application/octet-stream")
+    }
+    contentType
+  }
+
+  private val fileExtensionMap: util.Map[String, String] = new java.util.HashMap().asInstanceOf[java.util.Map[String, String]]
```

Comment:
This should not part of the code. Find the right java library will give the extension type rather than we are adding it.

Sample: 
https://www.baeldung.com/java-file-extension

Reply:
Used the Tika now.